static/app/views/profiling/content.tsx

```diff
@@ -277,11 +278,20 @@ function ProfilingContentLegacy({location}: ProfilingContentProps) {
   );
 }
 
+function validateTab(tab: unknown): tab is 'flamegraph' | 'transactions' {
+  return tab === 'flamegraph' || tab === 'transactions';
+}
+
+function decodeTab(tab: unknown): 'flamegraph' | 'transactions' {
+  return validateTab(tab) ? tab : 'transactions';
```

Comment:
A comment explaining why we're choosing transactions as the default would be helpful here